rclcpp/include/rclcpp/node.hpp

```diff
@@ -586,6 +586,14 @@ class Node : public std::enable_shared_from_this<Node>
   void
   undeclare_parameter(const std::string & name);
 
+  /// Load a list of parameters from a yaml parameter file.
+  /**
+   * \param[in] yaml_name The name of the yaml file that needs to be loaded.
+   */
+  RCLCPP_PUBLIC
+  std::vector<rcl_interfaces::msg::SetParametersResult>
+  load_parameters(const std::string & yaml_filepath, const std::string & node_name_);
```

Comment:
aside the on-going discussion on how to implement, do we need to have 2nd argument `node_name_`? what about `load_parameters` takes `yaml file` and internally calls `get_fully_qualified_name()`. if `load_parameters` belong to the Node class, i think it should load the parameters from yaml file in the context of the `Node`?